server/routes/referrals/referralsController.ts

```diff
@@ -396,16 +395,16 @@ export default class ReferralsController {
         throw new Error('Attempting to view complexity level without service category selected')
       }
 
-      const serviceCategory = await this.interventionsService.getServiceCategory(
-        res.locals.user.token.accessToken,
-        referral.serviceCategoryId
-      )
+      const [serviceCategory, serviceUser] = await Promise.all([
```

Comment:
Out of interest, is much difference between `Promise.all` and multiple `await` lines?

Reply:
`Promise.all` will run them concurrently, whereas multiple `await`s will run sequentially.